docs/api-guide/pagination.md

```diff
@@ -84,32 +93,49 @@ Note that the `paginate_queryset` method may set state on the pagination instanc
 
 Let's modify the built-in `PageNumberPagination` style, so that instead of include the pagination links in the body of the response, we'll instead include a `Link` header, in a [similar style to the GitHub API][github-link-pagination].
 
-    class LinkHeaderPagination(pagination.PageNumberPagination):
-        def get_paginated_response(self, data):
-            next_url = self.get_next_link()
            previous_url = self.get_previous_link()
+```py
```

Comment:
Noted. I think we do that one differently as it needs to display on GitHub. The docs ones all just use javascript based language detection.